tests/WorkerTest.php

```diff
@@ -44,13 +45,18 @@ public function testGetHandler()
 
     public function testGetHandlerNotCallable()
     {
+        $this->setExpectedExceptionRegExp(
+            HandlerException::class,
+            '/The handler for `test` is invalid./'
```

Comment:
Good point, missed that we were requiring PHPUnit 5.4, so it does make sense to use `expectException()` instead of `setExpectedException()` et al.